quiz-app/index.html

```diff
@@ -4,10 +4,18 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>WebDev Quiz</title>
     <link rel="stylesheet" href="css/common.css">
+    <link rel="stylesheet" href="css/homepage.css">
 </head>
 <body>
-    
+    <div class="main-container">
+        <h1 class="homepage-header">
+            Test your 
+            <span>WEB DEV</span>
+            knowlege
+        </h1>
+        <a href="pages/category" class="btn-begin">Let's Begin</a>
```

Comment:
check href? I hope you want it to target  HTML?

Reply:
ack!